build.sbt

```
@@ -69,13 +69,35 @@ lazy val `lang-tests-js` = project
     testFrameworks += new TestFramework("utest.runner.Framework")
   )
 
-lazy val node = project.dependsOn(`lang-jvm`, `lang-testkit` % "test;test->test")
+lazy val node = project.dependsOn(`lang-jvm`)
 
-lazy val `grpc-server`    = project.dependsOn(node % "compile;test->test;runtime->provided")
-lazy val `ride-runner`    = project.dependsOn(node % "compile;test->test", `grpc-server`)
-lazy val `node-it`        = project.dependsOn(node % "compile;test->test", `lang-testkit`, `repl-jvm`, `grpc-server`)
-lazy val `node-generator` = project.dependsOn(node % "compile->test")
-lazy val benchmark        = project.dependsOn(node % "compile;test->test")
+lazy val `node-testkit` = project
+  .in(file("node/testkit"))
+  .dependsOn(`lang-jvm`, `lang-testkit` % "compile->compile", `node` % "compile->test")
+  .settings(
+    libraryDependencies ++=
+      Dependencies.test.map(_.withConfigurations(Some("compile"))) ++ Dependencies.qaseReportDeps ++ Dependencies.logDeps ++ Seq(
+        "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5"
+      )
+  )
+lazy val `node-tests` = project
+  .in(file("node/tests"))
+  .dependsOn(`lang-jvm`, `node`, `lang-testkit` % "test;test->test", `node-testkit`)
+  .settings(libraryDependencies ++= Dependencies.node.value)
+
+lazy val `grpc-server` =
+  project.dependsOn(node % "compile;test->test;runtime->provided", `node-testkit`, `node-tests` % "compile->test")
```

1. `node` hasn't tests now, so we can remove `test->test` from dependencies like `node % "anything;test->test;anything"` (`anything` can mean nothing too).
2. Also, I think the `compile` configuration of `grpc-server` should **not** depend on the `test` configuration of `node-tests`, because it will contain test-related classes then (in the artifact).
```suggestion
lazy val `grpc-server` = project.dependsOn(node % "compile;runtime->provided", `node-testkit`, `node-tests` % "test->test")
```

---

Thank you, replaced.